nyc_buildings/nyc_buildings.ipynb

```diff
@@ -71,11 +118,26 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "cats = list(ddf.type.value_counts().compute().iloc[:10].index.values) + ['unknown']\n",
-    "ddf['type'] = ddf.type.replace({None: 'unknown'})\n",
-    "ddf = ddf[ddf.type.isin(cats)]\n",
-    "ddf['type'] = ddf['type'].astype('category').cat.as_known()\n",
+    "cats = list(ddf['type'].value_counts().compute().iloc[:10].index.values) + ['unknown']\n",
     "\n",
+    "ddf['type'] = ddf['type'].replace({None: 'unknown'})\n",
+    "ddf = ddf[ddf['type'].isin(cats)]\n",
+    "ddf['type'] = ddf['type'].astype('category').cat.as_known()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "SpatialPandas lets us build a spatial index for accessing spatially organized regions more quickly, so let's do that:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
Unfortunately, we have to remove the progress bar as it messes up the display on the website. Or find out if there's a way to configure something (the progress bar, sphinx/mystnb) to work around this. And if not, reproduce the issue and report it upstream.

<img width="1288" alt="image" src="https://github.com/user-attachments/assets/7b9827ce-61e5-4d1c-930b-3a3f15c0bd56">

Reply:
Removed.

Reply:
@Azaya89 did you also remove `ddf = ddf.build_sindex().persist()`? If yes, that is not correct, only the progress bar should be removed.

Reply:
Oops!